src/Health/Indicator/ApiHealthIndicator.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace Actuator\Health\Indicator;
+
+use Actuator\Health\HealthBuilder;
+use Guzzle\Http\Message\Request;
+use Guzzle\Http\Message\Response;
+
+class ApiHealthIndicator extends AbstractHealthIndicator
+{
+    /**
+     * @var Request
+     */
+    private $request;
+
+    public function __construct(Request $request)
+    {
+        $this->request = $request;
+    }
+
+    /**
+     * Actual health check logic.
+     *
+     * @param HealthBuilder $builder
+     * @throws \Exception any Exception that should create a Status::DOWN
+     * system status.
+     */
+    protected function doHealthCheck(HealthBuilder $builder)
+    {
+        /** @var Response $response */
+        $response = $this->request
+            ->send();
+
+        $builder->withDetail('statusCode', $response->getStatusCode());
+
+        if (!$response->isSuccessful()) {
+            $builder->down();
```

Comment:
If it's down, do you want to add the body to the details?  Maybe that will shed light on why it's down.  Something like:

```php
$builder->withDetail('message', $response->getBody());
```

Not sure if the syntax is right, but you get the idea.